src/openms/source/ANALYSIS/OPENSWATH/SwathWindowLoader.cpp

```diff
@@ -114,21 +117,24 @@ namespace OpenMS
     std::vector<double> & swath_prec_upper_ )
   {
     std::ifstream data(filename.c_str());
-    std::string line;
-
-    // check for presence of header, a header is present if the first line contains alphabetic characters
+    String line;
+    std::vector<String> headerSubstrings;
     double lower, upper;
-    std::getline(data, line); // possible header (or first line)
-    if (std::any_of(line.begin(), line.end(), [](char const &c) { return std::isalpha(c); }))
-    { // if any of the characters are alphabetical then there is a header
-      std::cout << "Read Swath window header: '" << line << "'\n";
-      std::getline(data, line); // first line of swath window
-     }
-    else 
+
+    // Check for presence of header
+    std::getline(data, line);
+    try { // If string can be successfully converted to double (excluding initial spaces) then the first line is not a header
+      StringUtils::split(line.trim(), ' ', headerSubstrings);
```

Comment:
in OpenMS we mainly use tab separated files not space separated.
Would it make sense to make this consistent here?
or allow both?

Reply:
The test file seems to use space separation (as oppose to tabs) so I think we should keep the space separation. I tried to address this issue here https://github.com/OpenMS/OpenMS/pull/6077/commits/10064a21c36cfe3b5e1de57c68b84b121612948a